src/runtime_src/core/include/xrt/xrt_graph.h

```diff
@@ -81,6 +82,9 @@ class graph
   graph(const xrt::device& device, const xrt::uuid& xclbin_id, const std::string& name,
         access_mode am = access_mode::primary);
 
+
+  graph(const xrt::hw_context& ctx, const std::string& name,
```

Comment:
Since this is a public API and may show up in doxygen comments, please add a brief comment along with description of arguments.